csvcubed/tests/unit/readers/test_preset_column_templates.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from csvcubed.readers.V1_0.preconfiguredtemplates import apply_preconfigured_values_from_template
+
+
+def test_func_accepts_dict_with_template():
+    column_config = {
+        "label" : "year",
+        "from_template" : "year"
+    }
+    apply_preconfigured_values_from_template(column_config, "V1_0")
+    assert column_config.get("column_uri_template") == "http://....../{+<column_name>}"
+
+def test_func_accepts_dict_without_template():
+    column_config = {
+        "label" : "year",
+    }
+    assert "label" in column_config
+    assert "column_uri_template" not in column_config
+    assert (apply_preconfigured_values_from_template(column_config, "v1_0")) == None
```

Comment:
This one should raise an exception IMO since it can't find the location of the lookup JSON file (lower-case `v`).

Reply:
I suppose it doesn't need to fetch the file if it doesn't have a template.